app/controllers/car_card_controller.rb

```diff
@@ -0,0 +1,9 @@
+class CarCardController < ApplicationController
```

Comment:
the name of the controller should be the same as the model

Reply:
Done